doc/releases/migration-guide-4.2.rst

```diff
@@ -133,6 +133,17 @@ Bluetooth Host
 Networking
 **********
 
+* The struct :c:struct:`net_linkaddr_storage` has been renamed to struct
```

Comment:
This struct doesn't exist anymore, so I don't think this directive actually works.

```suggestion
* The struct ``net_linkaddr_storage`` has been renamed to struct
```

Reply:
Good point, will fix that.